portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,86 @@
+<!-- 
+MAIN HTML
+command week 1: teachme ~/software-product-sprint/walkthroughs/week-1-web-development/portfolio-walkthrough.md
+command week 2: teachme ~/software-product-sprint/walkthroughs/week-2-server/project-1-walkthrough.md
+portfolio : cd \ ~/software-product-sprint/portfolio
+random message: teachme \
+    ~/software-product-sprint/walkthroughs/week-2-server/project-1-walkthrough.md
+
+https://nnietovega-sps-spring21.uc.r.appspot.com
+execute: mvn package exec:java
+href="https://github.com/naominietov"
+href="https://www.linkedin.com/in/naominietov"
+-->
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title> Naomi's Portfolio </title>
```

Comment:
I don't think it's particularly common to including spaces around the title. Is there an intention in doing that here?